### DISCUSSION TITLE
Ebike charging stations

### DISCUSSION THREAD

**Discussion by patrick-keogh** [osmandapp/OsmAnd#20631] [Ideas]
When displaying the cycling map (type : bicycle) I believe that POIs of category Charging station should only display if they have attribute Bicycle access : yes.

After all, that is what you are most likely interested in! You could still find car chargers using Search.

**Comment by mikehgentry** [comment]
Related #6136

The way to do it at the moment isn't through the POI overlay directly, but creating a custom filter (in the search menu) with whatever you want to search for, saving it with the buttons in the top right (it's a bit hidden), then using a Quick Action to enable that filter.

**Reply by patrick-keogh** [reply]
Yes I understand that. However if you select cycling map you automatically get lots of changes to the map presentation that relate to cycling, this would just be one more, for the sake of consistency.

Also the ergonomics of doing it that way is far more cumbersome than having the bicycle access:yes ones stand out as you scroll the map (just like bike routes do).
